content/windows.md

```diff
@@ -430,43 +220,84 @@ The next page will ask what type of installation you want to perform. Since ther
+
+Installing Windows to a **Dedicated Drive**
+
+1. Select the drive for Windows installation.
+    - This drive likely has 1 partition listed, or shows as "Unallocated Space"
+    - If there are more partitions listed, see the [Managing Partitions](#managing-partitions) section below for instructions on removing them.
+2. Click `Next` to install Windows to the drive.
+3. Proceed to [Completing the Installation](#completing-the-installation)
+
+Installing Windows to a **Shared Drive**
 
-Select the partition that matches the size and position in the list of what you created earlier using GParted. (Windows may select this partition automatically.) Click `Next` to install Windows to the partition.
+We are going to install Windows to an existing partition on the shared drive.
+
+1. Select the partition that matches what you created earlier using GParted.
+2. Click `Next` to install Windows to the partition.
+3. Proceed to [Completing the Installation](#completing-the-installation)
 
 >**Warning:** You should not need to format any partitions during this step. If you see a `Windows can't be installed on drive # partition #` message, either you have the wrong partition selected, or you did not select `NTFS` as the partition type when creating the partition in GParted earlier. Try selecting a different partition, or go back to [Shrinking Pop!_OS](#shrinking-pop_os).
 
-![Windows selecting a partition](/images/dual-booting/windows-select-partition.jpg)
+Installing **Windows Only**
+
+1. Select the drive you would like to install Windows on (likely the only drive)
+    - If there are partitions already on the drive, see the [Managing Partitions](#managing-partitions) section below for instructions on removing them.
+2. Click `Next` to install Windows to the partition.
+3. Proceed to [Completing the Installation](#completing-the-installation)
+
+#### Managing Partitions
 
-Next, wait for the Windows installation to complete. The installer will show progress as it copies and expands the OS files. The system may reboot one or more times.
+If the drive to be used for Windows already contains any partitions, start by deleting those partitions. To do this, you'll need to identify which drive will be used for Windows.
+
+For example, if you purchased a System76 machine with multiple drives, any extra drives should have a single partition (while the OS drive for Pop!_OS will have three or four partitions.) In this example, the Pop!_OS drive is 256GB and the Windows drive is 128GB, so we'll delete the single partition on `Drive 1` because it's closest to 128GB:
+
+![Windows deleting existing partitions](/images/dual-booting/windows-delete-existing-partitions.jpg)
+
+(If your flash drive is large enough, it may also show up in the list of drives; once again, identify it by its size and do not attempt to delete its partitions, since it's where the installer is stored.)
```

Comment:
This type of supplementary info can be placed in an informational block: 

> ℹ️ If your flash drive is large enough, it may also show up in the list of drives; once again, identify it by its size and do not attempt to delete its partitions, since it's where the installer is stored.